### ISSUE TITLE
Restore DMCA'd repository

### ISSUE THREAD

**Issue by Starz0r** [blackjack4494/yt-dlc#255]
Hi. I'm currently the maintainer for [youtube-dlc](https://chocolatey.org/package/youtube-dlc) package on Windows through Chocolatey. Currently, I'm having trouble getting new versions verified because my package still points to the old repository, and I'd like to get this fixed.

GitHub is currently offering a process to get the DMCA'd repositories backup and running, so if it is possible to do that still, that would be great. I imagine you could merge the old repository into the new one, causing the old URL to at least redirect, causing me less issues.

Thank you!

**Comment by blackjack4494** [comment]
I already sent a request a long time ago. I even sent a 2nd mail but they just replied that they are a bit busy at the moment. Hopefully this will be resolved very soon. Also having major troubles here as I am basically without any useful internet connection for most of the time (kudos to my provider....) since at least 2 weeks.

**Comment by nicolaasjan** [comment]
Read on Reddit:
https://reddit.com/r/youtubedl/comments/jv7vqb/youtubedls_repository_has_been_restored/gci1edu/

> Forks are not automatically being reinstated. Fork owners must re-fork from the parent repository. This is per an official email from GitHub Support.

**Comment by blackjack4494** [comment]
> Read on Reddit:
> https://reddit.com/r/youtubedl/comments/jv7vqb/youtubedls_repository_has_been_restored/gci1edu/
> 
> > Forks are not automatically being reinstated. Fork owners must re-fork from the parent repository. This is per an official email from GitHub Support.

Yes that is true. However as soon as they sent out that notification I sent out a request for it to be reinstated as seen in the screenshot.

**Comment by blackjack4494** [comment]
Update on this:
https://github.com/blackjack4494/youtube-dlc
It is currently back. However still need approval of github that applied changes fulfill their requirements. But should be fine.

**Comment by nicolaasjan** [comment]
> Update on this:
> https://github.com/blackjack4494/youtube-dlc
> It is currently back. However still need approval of github that applied changes fulfill their requirements. But should be fine.

Great!

But now we have 2 repo's.
If approved, will they somehow be combined into one?
